public/docs/_examples/intro/dart/lib/hero_list_component.dart

```diff
@@ -0,0 +1,25 @@
+library developer_guide_intro.hero_list_component;
+
+import 'package:angular2/angular2.dart';
+import 'package:developer_guide_intro/hero.dart';
+import 'package:developer_guide_intro/hero_detail_component.dart';
+import 'package:developer_guide_intro/hero_service.dart';
+
+@Component(
+    selector: 'hero-list',
+    templateUrl: 'hero_list_component.html',
+    directives: const [CORE_DIRECTIVES,HeroDetailComponent],
```

Comment:
Be sure to run the formatter on this (and all the other .dart files). I'm pretty sure there should be a space after the comma.